tests/connector/tests/apps_for_runtime_test.go

```diff
@@ -0,0 +1,96 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+	"github.com/kyma-incubator/compass/tests/pkg/certs"
+	"github.com/kyma-incubator/compass/tests/pkg/clients"
+	"github.com/kyma-incubator/compass/tests/pkg/fixtures"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	ScenariosLabel  = "scenarios"
+	TestScenario    = "test-scenario"
+	DefaultScenario = "DEFAULT"
+)
+
+func TestAppsForRuntimeWithCertificates(t *testing.T) {
+	scenarios := []string{DefaultScenario, TestScenario}
+	fixtures.UpdateScenariosLabelDefinitionWithinTenant(t, ctx, directorClient.DexGraphqlClient, cfg.Tenant, scenarios)
+	defer fixtures.UpdateScenariosLabelDefinitionWithinTenant(t, ctx, directorClient.DexGraphqlClient, cfg.Tenant, []string{DefaultScenario})
+
+	appIdToCommonName := make(map[string]string)
+
+	// Register first Application
+	firstApp, err := fixtures.RegisterApplicationFromInput(t, ctx, directorClient.DexGraphqlClient, cfg.Tenant, graphql.ApplicationRegisterInput{
+		Name:   "test-first-app",
+		Labels: map[string]interface{}{ScenariosLabel: []string{TestScenario}},
+	})
+	defer fixtures.CleanupApplication(t, ctx, directorClient.DexGraphqlClient, cfg.Tenant, &firstApp)
```

Comment:
It's a minor,but could you please do:
```
defer func(){
  fixtures.UnassignApplicationFromScenarios(t, ctx, directorClient.DexGraphqlClient, cfg.Tenant, firstApp.ID, true)
  fixtures.CleanupApplication(t, ctx, directorClient.DexGraphqlClient, cfg.Tenant, &firstApp)
}
```
This way it is easier to track the execution of the defer statements

Reply:
Regarding adding the test in the director ones:

Initially, I wanted to add it there because it is more intuitive. However, as this test requires issuing of certificates and all current director tests do not, the whole "setup" from the connector tests should have been copy-pasted to the director. 

I thought this was unnecessary overhead for a single test and that it will fit better in the connector for the certificate related stuff. In addition, nothing major was added/changed to the connector tests setup to make this test work, so I decided to leave it there.